main.go

```diff
@@ -0,0 +1,11 @@
+// This file only exists to make sure our API packages compile properly.
+package main
+
+import (
+	"github.com/kubevirt/kubevirt-ssp-operator/pkg/apis/kubevirt/v1"
+)
+
+func main() {
+	x := v1.KubevirtCommonTemplatesBundle{}
+	x = x
```

Comment:
Not sure what's the purpose of this line

Reply:
Without it golang will complain that x is not used anywhere.

Reply:
Ah, I missed that, will remove it

Reply:
@ksimon1 can't `x` be replaced with `_`?

Reply:
That is exactly what I did